lib/problem_check/twitter_login.rb

```diff
@@ -2,19 +2,22 @@
 
 class ProblemCheck::TwitterLogin < ProblemCheck
   self.priority = "high"
-
-  # TODO: Implement.
   self.perform_every = 24.hours
+  self.max_blips = 3
 
-  def call
+  def call(tracker)
     return no_problem if !authenticator.enabled?
     return no_problem if authenticator.healthy?
 
+    authenticator.disable if disable_failing_social_logins? && self.class.max_blips >= tracker.blips
```

Comment:
What if Twitter is the only configured login method? IMO in that case we should leave it enabled.

Reply:
We can I guess...but it seems strange that we can allow only one login method to be configured, which is one that can break based on a third party change or an API key. Shouldn't we at least require 2 to be configured? IDK how it's set up now but it just seems strange to me. Like if Twitter is their only method, and Twitter logins break...how do they get into the site?

Reply:
It's a bit of a "gg" problem to have. Since admins potentially can't login to update the Twitter credentials? 😅

Reply:
Yes that's what I was meaning...even before this change, admins are screwed if their only login method breaks right?